Lection15_HW/hw1.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+
+def test_get_request():
+    url = "https://api.punkapi.com/v2/beers/8"
+    expected_status_code = 200
+    expected_name = 'Fake Lager'
+    expected_abv = 4.7
+    response = requests.get(url)
+    assert response.status_code == expected_status_code, \
+        f"Status code from URL '{url}' is not {expected_status_code}"
+    assert response.json()[0]['name'] == expected_name
+    assert response.json()[0]['abv'] == expected_abv
+
+
+def test_delete_request():
+    url = "https://api.punkapi.com/v2/beers/8"
```

Comment:
You can reuse this url. In our situation it's the same, se we can add one variable.
For other situations in the future - create baseurl param and use it how you want (ex. baseurl + your_endpoint)